### ISSUE TITLE
App entry for 10524 not linked properly

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/faust-gen-html#356]
From here
http://dev.faustedition.net/print/app
I have a link to
http://dev.faustedition.net/print/faust.38#app.10523.eristbehendreistallesmitsichfort
But I get to the beginning of the scene.
And here:
http://dev.faustedition.net/print/faust.38#l10524
I have no link the list of app entries.

**Comment by thvitt** [comment]
The latter is #313 

Why the link points to 10523 and not 10524 is unclear to me

**Comment by thvitt** [comment]
```xml
  <app n="10523">
    <f:ins n="10523" place="after"><l n="10524">Er ist behend, reist alles mit sich fort.</l></f:ins>
    <ref>10524</ref>
    <!--Er <gap reason="ellipsis"/> fort. <i>none</i>-->
    <lem>Er <gap reason="ellipsis"/> fort.</lem>
    <!-- Er <gap reason="ellipsis"/> fort <i>erg G 2_IV_H.18 (ohne abschließenden Punkt) fehlt IV_H.13 H [type=2b]</i>-->
    <rdg wit="2_IV_H.18" hand="G" type="type_2b">Er <gap reason="ellipsis"/> fort <note>erg <seg type="hand">G</seg> <wit wit="2_IV_H.18">2_IV_H.18</wit> (ohne abschließenden Punkt) fehlt IV_H.13 H</note></rdg>
  </app>
```

**Comment by gerritbruening** [comment]
The link may point to 10523 because the whole of line 10524 is inserted by means of an app2norm line which points to 10523.

**Comment by thvitt** [comment]
yes, pointing to 10523 is correct. I think the easiest solution for this would be to move the `<note type="textcrit"/>` inside v. 10523

**Comment by gerritbruening** [comment]
Well now that the inserted line is inserted, it would be more logical to put the apparatus entry in this inserted line.
Don't know how much effort this takes, though.

**Comment by thvitt** [comment]
maybe just copy every apparatus entry that doesn't live inside a line into the preceding/following line and let the duplicate detection sort it out

so we'd need an extra step before the (current) pass2

probably best to first implement a validation step that detects app entries that aren't inside something that hasvars()

**Comment by thvitt** [comment]
JFTR validation results before the fix to this issue and #313 

Referenz | @n | f:ins/@place | erstes eingefügtes Element | @n davor | @n danach
-- | -- | -- | -- | -- | --
5441–5456 | 5441 | enclosing-lg | lg |   | 5441
5441–5456 | 5445 | enclosing-lg | lg |   | 5441
5441–5456 | 5449 | enclosing-lg | lg |   | 5441
5441–5456 | 5452 | enclosing-lg | lg |   | 5441
5484–5493 | 5484 | before | milestone | before_5484 |  
10524 | 10523 | after | l 								n=10524 | 10524 |  
vor 10811 | before_10811_a | after | stage 								n=before_10811_b | before_10811_b | 10811

**Comment by gerritbruening** [comment]
Terrific!
